listener/changes.go

```diff
@@ -101,15 +98,22 @@ func (cl *ChangesListener) SetupDirectoriesToWatch(directory string) {
 
 func (cl *ChangesListener) EventHandler(event fsnotify.Event) bool {
 	if isModifiedFile(event) {
-		fmt.Println(cl.job.ExecuteJob())
+		cl.jobRunner.RunJobs()
 		return true
 	}
 
 	return false
 }
 
-func isHiddenFile(fileName string) bool {
-	return filepath.HasPrefix(fileName, ".") && fileName != "." && fileName != ".."
+func isHiddenFile(path string) bool {
+	absolutePath, err := filepath.Abs(path)
+	if err != nil {
+		log.Panic(err)
+	}
+
+	fileName := filepath.Base(absolutePath)
+
+	return filepath.HasPrefix(absolutePath, ".") || (fileName[0:1] == "." && fileName != "." && fileName != "..")
```

Comment:
```suggestion
	return filepath.HasPrefix(absolutePath, ".") || (fileName[0:1] == ".")
```

Acho que não precisa disso porque o absolute já vai resolver o relative path

Reply:
Só se realmente o nome for `.` ou `..` ai o cara é um cabaço de fazer isso